econml/dml/causal_forest.py

```diff
@@ -564,6 +857,92 @@ def shap_values(self, X, *, feature_names=None, treatment_names=None, output_nam
                                                   background_samples=background_samples)
     shap_values.__doc__ = LinearCateEstimator.shap_values.__doc__
 
+    def ate__inference(self):
+        """
+        Returns
+        -------
+        ate__inference : NormalInferenceResults
+            Inference results information for the `ate_` attribute, which is the average
+            constant marginal treatment effect of each treatment for each outcome, averaged
+            over the training data and with a doubly robust correction.
+            Available only when `discrete_treatment=True` and `oob=True`.
+        """
+        return NormalInferenceResults(d_t=self._d_t[0] if self._d_t else 1,

```

Comment:
This and the subsequent properties are also showing as uncovered by tests.

Reply:
This is weird! This should be covered!